llvm/test/CodeGen/AArch64/arm64-shrink-wrapping.ll

```diff
@@ -1036,15 +1036,27 @@ false:
 define void @stack_realign2(i32 %a, i32 %b, ptr %ptr1, ptr %ptr2, ptr %ptr3, ptr %ptr4, ptr %ptr5, ptr %ptr6) {
```

Comment:
Are we? It seems the only change is that blocks were moved around.

Reply:
The before version has the STPs which save x19-x30 in the entry block, so they were always executed, and the conditional branch only skipped over two instructions. In the after version, the LDP/STP sequences are moved into a different basic block, and the conditional branch skips them completely.

The generated code looks correct to me - it now uses x18 for re-aligning the stack, instead of x9, and x18 is otherwise unused. If this is an expected change cause by `LiveRegUnits` doing a better job than `LivePhysRegs` then it's not a problem, the comment just needs updating.